_sass/custom/_accordion.scss

```diff
@@ -0,0 +1,8 @@
+.content-constrained {
```

Comment:
Please import this file, because I had to remove from the previous pull request.